README.md

```diff
@@ -1,3 +1,89 @@
-# gz-beat-shipper
+# beatshipper
 
 Since there’s no way to send GNU zip files through filebeat, this service will be responsible for checking if there are new .gz files based on a path that we’ll explode using globbing, decompress them, send them using the filebeat service with the provided configuration and store them in a local file registry.
+
+It can be used also for not compressed files.
+
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+## Requirements
+
+[Download](https://golang.org/dl/) and install **Go**.
+
+> 🔔 Please note: version 1.18 or higher required
+
+## Installation
+
+- Move to the cloned repository and install with:
+
+```
+$ go install
+```
+
+```
+$ beatshipper -h
+Sends data based on paths that will be exploded using GLOB
+with the possibility of passing GNU Zip files also.
+
+Usage:
+  beatshipper [command]
+
+Available Commands:
+  completion  Generate the autocompletion script for the specified shell
+  help        Help about any command
+  send        Send the message as a beat
+
+Flags:
+  -h, --help      help for beatshipper
+  -v, --version   version for beatshipper
+
+Use "beatshipper [command] --help" for more information about a command.
+```
+
+## Configuration
+
+We should create the configuration in the following file `/etc/beatshipper/beatshipper-conf.yml`. An example is located in the root of the project. We should add the following fields:
+
+- `host`
+- `port`
+- `path` array of paths that will be exploded using `GLOB`
+- `registry`: the name of the file where we'll store the processed files.
+- `recheck` time of recheck (re-process)
+
```

Comment:
Have these as comments on the config file as well.

Reply:
Changed, thank you.